tests/test_shortcuts.py

```diff
@@ -199,6 +199,12 @@ def test_run_show_cmd_logfile_stdout(self, mock_out):
         self.assertEqual(mock_out.getvalue(),
                          'COMMAND: echo foo\n-----------------\nfoo\n')
 
+    def test_run_split_in_middle_of_utf8_sequence(self):
+        logfile = os.path.join(self.tmp_dir, 'output.log')
+        cmd = "printf ' ' && printf 'č%.0s' {1..10000}"
```

Comment:
Apparently this isn't working properly in Travis because that {1..10000} syntax is a feature of bash, not supported by the dash shell used by Debian & Ubuntu.

Reply:
Ah, that makes sense. I'll rework to avoid that then.